example_app_generator/generate_app.rb

```diff
@@ -47,6 +47,11 @@
     |  gem 'mime-types', '< 3'
     |end
     |
+    |# Version 6.0.4 of rdoc requires Ruby 2.2
+    |if RUBY_VERSION < '2.2'
+    |  gem 'rdoc', '< 6.0.4'
```

Comment:
It's better to do a gsub, to specify the version only if rails adds it, and ensures theres only one, see line 24